dev/com.ibm.ws.rest.handler.validator_fat/fat/src/com/ibm/ws/rest/handler/validator/fat/ValidateJCATest.java

```diff
@@ -335,7 +335,7 @@ public void testMultipleConnectionFactories() throws Exception {
         //stack = j.getJsonArray("stack");
         //assertNotNull(err, stack);
         //assertTrue(err, stack.size() > 10); // stack is actually much longer, but size could vary
-        //assertTrue(err, stack.getString(0).startsWith("org.test.validator.adapter.ManagedConnectionFactoryImpl.createManagedConnection(ManagedConnectionFactoryImpl.java:"));
+        //assertTrue(err, stack.getString(0).startsWith("org.test.validation.adapter.ManagedConnectionFactoryImpl.createManagedConnection(ManagedConnectionFactoryImpl.java:"));
```

Comment:
This is commented out so it probably doesn't matter, but you updated the package name here without actually updating a package name of the resource adapter.